### DISCUSSION TITLE
table渲染之后，所有列挤到一起

### DISCUSSION THREAD

**Discussion by yaoyunnuo** [layui/layui#2398] [问答]
在后台管理模板中，点击左侧菜单，右侧以页签方式打开页面，使用的是iframe,当连续点击左侧菜单，打开多个页面时，处于隐藏状态的下的页签，如果页面
内部有table，很多时候table渲染出来列都挤到了一起，有遇到这种问题的吗？

![微信截图_20241213172352](https://github.com/user-attachments/assets/6098d906-6ada-442b-b835-68157b91f65e)

**Comment by Sight-wcg** [comment]
#2370

**Reply by Sight-wcg** [reply]
下个版本修复这类问题
